easybuild/easyconfigs/c/Calcam/Calcam-2.1.0-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonPackage'
+
+name = 'Calcam'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://euratom-software.github.io/calcam'
+
+description = """Calcam is a Python package providing tools for spatial calibration
+ of cameras, i.e. determining the mapping between pixel coordinates in an image and
+ real-world 3D sight lines & coordinates."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/euratom-software/calcam/archive']
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'download_filename': 'v%(version)s.tar.gz',  # provided source tarball is not versioned...
+}]
+checksums = ['8348545719209650ac62ed8c77087ec6e42df77995ead59f3b2e035da962a679']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('matplotlib', '2.1.2', '-Python-%(pyver)s'),
+    ('OpenCV', '3.4.1', '-Python-%(pyver)s'),
+    ('PyQt5', '5.9.2', '-Python-%(pyver)s'),
+    ('VTK', '8.1.0', '-Python-%(pyver)s'),
+]
+
+use_pip = True
+
+download_dep_fail = True
+
+preinstallopts = 'echo n |'
```

Comment:
@SimonPinches What's this about? Is the `pip install` asking questions?!

Reply:
Yes, `pip` asks whether you want to carry out a conversion from earlier representations resulting in a crash without this line as follows:
```
running install_egg_info                                                                                                                                                                                                                running egg_info                                                                                                                                                                                                                        creating Calcam.egg-info                                                                                                                                                                                                                writing requirements to Calcam.egg-info/requires.txt                                                                                                                                                                                    writing Calcam.egg-info/PKG-INFO                                                                                                                                                                                                        writing top-level names to Calcam.egg-info/top_level.txt                                                                                                                                                                                writing dependency_links to Calcam.egg-info/dependency_links.txt                                                                                                                                                                        writing entry points to Calcam.egg-info/entry_points.txt                                                                                                                                                                                writing manifest file 'Calcam.egg-info/SOURCES.txt'                                                                                                                                                                                     reading manifest file 'Calcam.egg-info/SOURCES.txt'                                                                                                                                                                                     writing manifest file 'Calcam.egg-info/SOURCES.txt'                                                                                                                                                                                     Copying Calcam.egg-info to /work/imas/opt/EasyBuild/software/Calcam/2.1.0-intel-2018a-Python-2.7.14/lib/python2.7/site-packages/Calcam-2.1.0-py2.7.egg-info                                                                             running install_scripts                                                                                                                                                                                                                 Installing calcam script to /work/imas/opt/EasyBuild/software/Calcam/2.1.0-intel-2018a-Python-2.7.14/bin                                                                                                                                writing list of installed files to '/tmp/eb-Rk8w4C/pip-oAc8QF-record/install-record.txt'                                                                                                                                            
                                                                                                                                                                                                                                            Installaction complete.                                                                                                                                                                                                                 It appears you have a data directory from Calcam 1.x and might be upgrading to Calcam 2.                                                                                                                                                Would you like to run the Calcam 1 -> Calcam 2 file conversion tool now? [Y/n] Traceback (most recent call last):                                                                                                                   
      File "<string>", line 1, in <module>                                                                                                                                                                                                    File "/tmp/eb-Rk8w4C/pip-MwE3Yf-build/setup.py", line 169, in <module>                                                                                                                                                                    if query_yes_no('\nInstallaction complete.\nIt appears you have a data directory from Calcam 1.x and might be upgrading to Calcam 2.\nWould you like to run the Calcam 1 -> Calcam 2 file conversion tool now?'):               
      File "/tmp/eb-Rk8w4C/pip-MwE3Yf-build/setup.py", line 58, in query_yes_no                                                                                                                                                                 choice = raw_input().lower()                                                                                                                                                                                                        EOFError: EOF when reading a line
```

Reply:
@SimonPinches Please add a comment above this line that mention the question that `pip` is asking, and also explain what the point is of the `echo n |`, since it looks a bit weird.

Reply:
@boegel Done - see latest commit to this branch.